### ISSUE TITLE
Remove Pa11y Rules

### ISSUE THREAD

**Issue by PaleBluDot** [American-Technion-Society/email-templates#3]
Ignored these rules to fix some other issues but these should be updated across all templates. 

`pa11y.json`

```json
"ignore":  [
  "WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2",	 
  "WCAG2AA.Principle2.Guideline2_4.2_4_2.H25.1.NoTitleEl"
]
```
